iam_policy/templates/jenkins_cloudwatch_agent_integration.json.tpl

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Should this policy be renamed: `jenkins_cloudwatch_agent.json.tpl`. Looks like it will be used both for the integration and prod Jenkins instances?

Reply:
Good point, I've renamed it.